views/024_integrations.sql

```diff
@@ -0,0 +1,4 @@
+CREATE OR REPLACE VIEW integrations_todo AS
```

Comment:
We seem to already have a "integrations" table
https://github.com/flanksource/duty/blob/987e8c512cbc1235aa145f802c955608a6fb2a39/schema/system.hcl#L93-L139